frontend/packages/topology/src/components/modals/MoveConnectionModal.tsx

```diff
@@ -70,12 +74,12 @@ const MoveConnectionForm: React.FC<FormikProps<FormikValues> & MoveConnectionMod
 
   return (
     <form onSubmit={handleSubmit} className="modal-content modal-content--no-inner-scroll">
-      <ModalTitle>Move Connector</ModalTitle>
+      <ModalTitle>{t('topology~Move Connector')}</ModalTitle>
       <ModalBody>
         <Title headingLevel="h2" size="md" className="co-m-form-row">
-          Connect
+          {t('topology~Connect')}
           <strong>{` ${edge.getSource().getLabel()} `}</strong>
-          to
+          {t('topology~to')}
```

Comment:
This should be one translation otherwise it won't come out properly in other languages.